packages/config/src/discovery/OpStackTypes.ts

```diff
@@ -46,6 +47,10 @@ export const OP_STACK_CONTRACT_DESCRIPTION: OPStackContractTemplate[] = [
     coreDescription:
       "The {0} (L1xDM) contract sends messages from L1 to L2, and relays messages from L2 onto L1. In the event that a message sent from L1 to L2 is rejected for exceeding the L2 epoch gas limit, it can be resubmitted via this contract's replay function.",
   },
+  {
+    name: 'L1StandardBridge',
+    coreDescription: "The {0} contract is the main entry point to deposit ERC20 tokens from L1 to L2. This contract can store any token.",
```

Comment:
```suggestion
    coreDescription: "The {0} contract is the main entry point to deposit ERC20 tokens from L1 to L2.",
```
<img width="858" alt="image" src="https://github.com/l2beat/l2beat/assets/30298476/65b419f8-762c-4d10-94af-f016c72e67bb">
The other sentence is added by some logic for escrows